src/pages/Workout.js

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from 'react';
+import { Container } from '../styles';
+import { Dropdown, Option } from '../components/Dropdown';
+import {
+  AnchorButton, Row,
+} from '../Components';
+import Header from '../components/Header';
+
+// This component will handle the diet lookup feature which will direct user to
+// the best overall Diets.
+export default function App() {
+  const [optionValue, setOptionValue] = useState('');
+  const [option, setOption] = useState('');
+  const [value, setValue] = useState('');
+  const [link, setLink] = useState();
+
+  const handleStretches = (e) => {
+    setOptionValue(e.target.value);
+    // eslint-disable-next-line no-console
+    console.log(e.target.value);
+    if (e.target.value === 'Biceps') {
+      setLink('https://musclewiki.com/Stretches/Male/Biceps');
+    }
+    if (e.target.value === 'Chest') {
+      setLink('https://musclewiki.com/Stretches/Male/Chest');
+    }
+    if (e.target.value === 'Abdominal') {
+      setLink('https://musclewiki.com/Stretches/Male/Abdominals');
+    }
+    if (e.target.value === 'Quads') {
+      setLink('https://musclewiki.com/Stretches/Male/Quads');
+    }
+    if (e.target.value === 'Calves') {
+      setLink('https://musclewiki.com/Stretches/Male/Calves');
```

Comment:
These `if` statements, and the similar statements below, would be better represented as a map or a switch statement. You can also use `selectedIndex` to get the links from an array.

Reply:
What do you mean by this are there any examples I could see ?

Reply:
I tried doing a switch case but i was confused as to what value i would be passing through the parameter. I’d think I’d rather use a map or see a mapping example.

Reply:
And all honesty this page wasn’t in presentation and it may not be worth adding this into final version it was just an extra page for lack of content.

Reply:
Up to you on whether to include it. Either way, a switch statement would switch on `e,target,value` with the strings as cases and a map would look something like this:

```js
const STRETCHES = {
    Biceps: 'https://musclewiki.com/Stretches/Male/Biceps',
    Chest: ''https://musclewiki.com/Stretches/Male/Chest',
    ...
};

if (e.target.value in STRETCHES) setLink(STRETCHES[e.target.value]);
```

Reply:
I think i may close and delete the branch when i get time